.circleci/soltest.sh

```diff
@@ -47,15 +47,28 @@ mkdir -p test_results
 ulimit -s 16384
 
 get_logfile_basename() {
-    local filename="${EVM}"
-    test "${OPTIMIZE}" = "1" && filename="${filename}_opt"
-    test "${ABI_ENCODER_V2}" = "1" && filename="${filename}_abiv2"
+  local filename="${EVM}"
+  test "${OPTIMIZE}" = "1" && filename="${filename}_opt"
+  test "${ABI_ENCODER_V2}" = "1" && filename="${filename}_abiv2"
 
-    echo -ne "${filename}"
+  echo -ne "${filename}"
 }
 
-BOOST_TEST_ARGS="--color_output=no --show_progress=yes --logger=JUNIT,error,test_results/`get_logfile_basename`.xml ${BOOST_TEST_ARGS}"
-SOLTEST_ARGS="--evm-version=$EVM $SOLTEST_FLAGS"
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Were the semantic tests not run on osx before?

Reply:
Yes, the semantic tests where also executed on osx before.

In the past the finding of the evmone shared library was done implicitly. But this implicit searching for emvone is disabled, if a `--vm` argument is supplied. The shared libraries need then to be defined explicitly. The main reason for this is that we decided not only to support evmone or hera, the `--vm` parameter can be used to define any evmc compatible shared library.